tasks/celery.yml

```diff
@@ -39,25 +39,27 @@
 # A systemd-tmpfiles setting is set up here so that /var/run/celery is created at
 # boot time with permissions allowing the project_name user to store their PIDs
 # there.
-- name: install celery tmpfile.d pid directory setting
-  template: src=celery_tmpfile.conf
-            dest=/etc/tmpfiles.d/celery.conf
-            owner=root
-            group=root
-            mode=0600
-  register: celery_tmpfile
+- when: not subproject
```

Comment:
I have a couple of comments about this task.  The first doesn't directly have to do with this change at all: I see that the template file being used uses the `{{ project_name }}` variable for the resulting conf file.  Looking more closely at the semantics of tmpfiles.d conf files, I see that the positional arguments it is being used for is for a UID and GID, so `{{ project_user }}` should be what is used instead to avoid problems when a project has this variable configured separately from `project_name`.

The remainder of my commentary is this: the conf template used for this is super-small, and would probably be better suited as a `lineinfile` task instead.  I think doing so would remove the need for a `subproject` variable, which seems a bit kludgy to me.

Postscript: Due to the account removal task in tequila-common, some caution is warranted when considering `project_user` for a multiple-project-per-environment setup like the one you all are contemplating.  Perhaps the right thing to do is to strongly recommend configuring the `project_user` to be the same across all projects in such a circumstance?

Reply:
The ``subproject`` variable was going to be added in multiple tequila roles, as an optimization for multi-project environments so we don't repeat tasks for every project that really only need to be done once. 

``subproject``t feels kludgy to me too, though, and I hope someone comes up with a cleaner way to do this.